ui_framework/components/local_nav/_local_search.scss

```diff
@@ -44,6 +48,15 @@
     transform: translateY(-50%); /* 1 */
   }
 
+  .kuiLocalSearchAssistedInput__assistanceLink {
+    display: none;
+  }
+
+  .kuiLocalSearchInput:focus + .kuiLocalSearchAssistedInput__assistance .kuiLocalSearchAssistedInput__assistanceLink,
+  .kuiLocalSearchAssistedInput__assistanceLink:active {
+    display: inherit;
+  }
+
```

Comment:
You're right, that interaction stinks as well. Unless @snide has any good ideas, tomorrow I'll switch it back to showing all the time so this doesn't become a big time sink.